format-specs/geoparquet.md

```diff
@@ -110,26 +110,24 @@ The axis order of the coordinates in WKB stored in a geoparquet follows the de f
 (x, y) where x is easting or longitude and y is northing or latitude. This ordering explicitly overrides the axis order as specified in the CRS.
 This follows the precedent of [GeoPackage](https://geopackage.org), see the [note in their spec](https://www.geopackage.org/spec130/#gpb_spec).
 
-#### geometry_type
+#### geometry_types
 
-This field captures the geometry type(s) of the geometries in the
+This field captures the geometry types of the geometries in the
 column, when known. Accepted geometry types are: "Point", "LineString",
 "Polygon", "MultiPoint", "MultiLineString", "MultiPolygon",
 "GeometryCollection".
 
 In addition, the following rules are used:
 
 - In case of 3D geometries, a " Z" suffix gets added (e.g. "Point Z").
-- The value can be a single string or an array of strings in case multiple
-  geometry types are present (e.g. ["Polygon", "MultiPolygon"]).
-- Additionally the value "Unknown" is accepted to explicitly signal that the
-  geometry type is not known.
+- A list of values indicates that multiple geometry types are present (e.g. ["Polygon", "MultiPolygon"]).
```

Comment:
```suggestion
- A list of multiple values indicates that multiple geometry types are present (e.g. ["Polygon", "MultiPolygon"]).
```

? (since we are now always using a list, this otherwise is a bit misleading?)

Reply:
Sounds good.  Updated in the latest commit.